src/server/services/persist-upload.js

```diff
@@ -50,7 +50,7 @@ async function persistUpload ({ filename, user, buffer }) {
   return upload
 }
 
-function bufferForUpload (upload) {
+async function bufferForUpload (upload) {
```

Comment:
just curious, why is this necessary?

Reply:
I don't think it is, actually.

I was fiddling with things because I noticed that bufferForUpload was running in series instead of in parallel when called from `recordsForReportingPeriod`.  I'm still not 100% sure why, but inlining `extractRecords` to `recordsForUpload` seemed to do the trick.

I guess I left it in because otherwise there's nothing indicating that this function is async without reading more code.